bundle/deployplan/action.go

```diff
@@ -41,6 +41,10 @@ var ShortName = map[ActionType]ActionType{
 	ActionTypeUpdateWithID: ActionTypeUpdate,
 }
 
+func (a ActionType) IsNoop() bool {
+	return a == ActionTypeNoop || a == ActionTypeUnset
```

Comment:
Isn't `ActionTypeUnset` a bug?

Reply:
It is a bug. Probably makes sense to add an explicit err check for it.